src/migrations/20230828103020-program-personnel-cleanup.js

```diff
@@ -10,35 +10,29 @@ module.exports = {
       await prepMigration(queryInterface, transaction, sessionSig);
 
       await queryInterface.sequelize.query(`
-         /* 1. Backup all current program personnel records in a table */
-         SELECT * INTO "ProgramPersonnel_backup_20230828" FROM "ProgramPersonnel"
-
-
-         /* 2. Create a table of everything we want to keep */
-         DROP TABLE IF EXISTS "ProgramPersonnelToKeep";
-         CREATE TEMP TABLE "ProgramPersonnelToKeep" AS (
-            SELECT
-                 "firstName",
-                 "lastName",
-                 "role",
-                 "grantId",
-                 "programId",
-                 "active",
-                 count(id),
-                 min("id") AS "idToKeep"
-                 FROM "ProgramPersonnel"
-                 GROUP BY
-                 "firstName",
-                 "lastName",
-                 "role",
-                 "grantId",
-                 "programId",
-                 "active"
-                 order by "active" asc
+         /* 1. Remove records we don't want */
+         WITH initial_entries AS (
+         SELECT
+           "firstName",
+           "lastName",
+           "role",
+           "grantId",
+           "programId",
+           active,
+           MIN(id) AS first_insert
```

Comment:
This is not the best set of fields to match on, I saw some records that only changed email address, while name remained the same. and others where the name changed and email remained the same. I think email needs to be included for comparison.